### ISSUE TITLE
Slight errors

### ISSUE THREAD

**Issue by JackAtKitronik** [rhubarbdog/kitronik-gamezip64#1]
Some fun games, thanks for sharing!

In battleships, 'dead' was undefined on line 239, can be fixed by initialising dead above the start of the loop.

The Two Weeks game worked well and had no problems flashing it to the micro:bit in Mu Editor but had some errors in the new micro:bit Python editor to do with the 'display' functions.

**Comment by rhubarbdog** [comment]
Did you have the chance to try 2 player 2 weeks?

**Comment by JackAtKitronik** [comment]
> Did you have the chance to try 2 player 2 weeks?

Yes! We played with 3 players. It's a fun game. Makes great use of the micro:bit radio.

Although, I didn't feel like it was much of a Fortnight clone, much more of a maze game. You could try adding some enemies into the map that chase the players around and respawn the players if they get caught.

**Comment by rhubarbdog** [comment]
yes the problem seems to be ram because i tried adding enemies and could have 10 but as soon as i tried to make them move some where i was crashing glitching and hanging. i really wanted it to be a multi player environment over radio.
there seems to be enough code space to make it into a kind of world of war craft type game if the quest length was kept small

**Comment by rhubarbdog** [comment]
hi
fixed that bug plus one more in `battle-ships.py`
it's hard to do any meanigful testing with only one game zip. i have to simulate player 2 using a radio chat program i created in from my microbit-fun repository.
